cmake/Modules/FindUDev.cmake

```diff
@@ -29,9 +29,9 @@ IF (UDEV_LIBRARIES AND UDEV_INCLUDE_DIR)
     execute_process(COMMAND pkg-config --atleast-version=143 libudev RESULT_VARIABLE UDEV_STABLE)
     # retvale is 0 of the condition is "true" so we need to negate the value...
     if (UDEV_STABLE)
-	set(UDEV_STABLE 0)
+    set(UDEV_STABLE 0)
     else (UDEV_STABLE)
-	set(UDEV_STABLE 1)
+    set(UDEV_STABLE 1)
```

Comment:
The indentation was good before...
And I wouldn't change third-party copyrighted files if it's not necessary.